workflow-service/src/main/java/com/redhat/parodos/project/controller/ProjectController.java

```diff
@@ -63,7 +65,8 @@ public ProjectController(ProjectServiceImpl projectService) {
 			@ApiResponse(responseCode = "201", description = "Created",
 					content = { @Content(mediaType = "application/json",
 							schema = @Schema(implementation = ProjectResponseDTO.class)) }),
-			@ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content) })
+			@ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content),
+			@ApiResponse(responseCode = "409", description = "Conflict", content = @Content) })
```

Comment:
When it should return 409? Should we add a test for this?

Reply:
I'll address the exception handling from the rest-controller as part of introducing a global exception handler as part of https://issues.redhat.com/browse/FLPATH-342.
I didn't want to implement this within this patch, in order not to overload it with changes.
I don't mind excluding it from this PR.

Reply:
I removed the API changes and will address them in the next PRs